apps/chat/src/components/Files/PreUploadModal.tsx

```diff
@@ -409,9 +449,7 @@ export const PreUploadDialog = ({
           onClick={handleUpload}
           disabled={selectedFiles.length === 0}
         >
-          {customUploadButtonLabel
-            ? customUploadButtonLabel
-            : t('Upload and attach files')}
+          {customUploadButtonLabel ? customUploadButtonLabel : t('Upload')}
```

Comment:
Maybe need to `Upload and attach` instead?

Reply:
On the daily meeting it was decided to leave 'upload' here